controllers/cruisecontroloperation_controller.go

```diff
@@ -97,6 +100,12 @@ func (r *CruiseControlOperationReconciler) Reconcile(ctx context.Context, reques
 		return reconciled()
 	}
 
+	// Skip reconciliation for Cruise Control Status operation
+	if currentCCOperation.CurrentTaskOperation() == banzaiv1alpha1.OperationStatus {
```

Comment:
Not sure why did I need to add this, will double check and remove it if it doesn't make sense

Reply:
I verified and can confirm that everything works as expected without this section, so it's safe to remove it.

Basically the only reason why I added it is to avoid logging an error:

```
Koperator does not support this operation...
```
as I thought it can be misleading, instead I considered that it sounds better to have
```
skipping reconciliation for Cruise Control Status operation
```

In conclusion we can remove it if you think that it doesn't bring value, let me know what you think about this